### ISSUE TITLE
Problem: timing not implemented in the  staking ledgers GQL endpoint 

### ISSUE THREAD

**Issue by trevorbernard** [Granola-Team/mina-indexer#751]
This is required for the mina-pool-payout script

```gql
    query stakingLedger($ledgerHash: String, $delegate: String) {
        stakes(limit: 100000, query: { ledgerHash: $ledgerHash, delegate: $delegate }) {
            pk
            balance
            delegate
            timing {
                cliff_amount
                cliff_time
                initial_minimum_balance
                vesting_increment
                vesting_period
            }
        }
    }
```

**Comment by jhult** [comment]
[Here](https://github.com/jrwashburn/mina-pool-payout/blob/f41a8c8c2cccf1bb9d2fefc286084ce011bc1f74/src/core/dataProvider/dataprovider-minaexplorer/staking-ledger-gql.ts#L6-L21) is the relevant code from `mina-pool-payout`.

**Comment by jhult** [comment]
Fixed in #814
